zcash_client_backend/src/sync.rs

```diff
@@ -135,7 +135,7 @@ where
             "Refreshing UTXOs for {:?} from height {}",
             account_id, start_height,
         );
-        refresh_utxos(params, client, db_data, account_id, start_height).await?;
+        refresh_utxos(params, client, db_data, account_id, start_height, false).await?;
```

Comment:
Expand the comment to explain why it is correct to exclude ephemeral addresses.

Reply:
This sync code is in the process of being refactored by @str4d for inclusion in `zallet`; I made the minimal change to preserve the existing behavior here. This call will actually happen in a separate process that handles the retrieval queue (and will likely not even be a singular call for an account) so the goal here is minimizing disruption.

Reply:
This is no longer applicable given the changes to `get_transparent_receivers`.